{"name":"vote","hostname":"itxchy-xubuntu","pid":12120,"level":60,"msg":"Mongoose: production connection failed { mongoose: true,\n  err: \n   { MongoError: failed to connect to server [undefined:27017] on first connect\n       at Pool.<anonymous> (/home/itxchy/fcc/vote/node_modules/mongodb-core/lib/topologies/server.js:326:35)\n       at emitOne (events.js:96:13)\n       at Pool.emit (events.js:188:7)\n       at Connection.<anonymous> (/home/itxchy/fcc/vote/node_modules/mongodb-core/lib/connection/pool.js:270:12)\n       at Connection.g (events.js:291:16)\n       at emitTwo (events.js:106:13)\n       at Connection.emit (events.js:191:7)\n       at Socket.<anonymous> (/home/itxchy/fcc/vote/node_modules/mongodb-core/lib/connection/connection.js:175:49)\n       at Socket.g (events.js:291:16)\n       at emitOne (events.js:96:13)\n       at Socket.emit (events.js:188:7)\n       at connectErrorNT (net.js:1022:8)\n       at _combinedTickCallback (internal/process/next_tick.js:74:11)\n       at process._tickCallback (internal/process/next_tick.js:98:9)\n     name: 'MongoError',\n     message: 'failed to connect to server [undefined:27017] on first connect' } }","time":"2017-03-14T00:18:22.450Z","v":0}
